src/sentry/seer/similarity/utils.py

```diff
@@ -43,9 +44,12 @@ def get_stacktrace_string(data: dict[str, Any]) -> str:
     found_non_snipped_context_line = False
     result_parts = []
 
+    metrics.distribution("seer.grouping.exceptions.length", len(exceptions))
+
     # Reverse the list of exceptions in order to prioritize the outermost/most recent ones in cases
     # where there are chained exceptions and we end up truncating
-    for exception in reversed(exceptions):
+    # Limit the number of chained exceptions to 30
```

Comment:
Nit: Since the number is now controlled by a variable, leaving it out of the comment means we don't have to remember to update the comment if/when we update the value.

```suggestion
    # Limit the number of chained exceptions
```